dicom_csv/misc.py

```diff
@@ -1,22 +1,29 @@
 import os
 import warnings
+from functools import partial
 from operator import itemgetter
 from os.path import join as jp
+from typing import Optional
 
 import numpy as np
+from pydicom.pixel_data_handlers import convert_color_space
 
 from .utils import *
 
 __all__ = 'get_image', 'stack_images'
 
 
-def get_image(instance: Dataset):
+def get_image(instance: Dataset, to_color_space: Optional[str] = None):
     def _to_int(x):
         if x == int(x):
             x = int(x)
         return x
 
-    array = instance.pixel_array
+    if to_color_space is None or not hasattr(instance, 'PhotometricInterpretation'):
```

Comment:
if PhotometricInterpretation is not defined in this case - this should be an error. Now we're just silencing it